### ISSUE TITLE
Is this a way to init Occupancy grid by myself?

### ISSUE THREAD

**Issue by StarsTesla** [KAIR-BAIR/nerfacc#250]

**Comment by Abc11c** [comment]
Not sure if this is the same, Is there a way to get density/ coarse depth from `ray_intersection/ travese_grid` func? Given I have the binaries/ voxel information as to where the object is already present? 

Thanks!

**Comment by RobotDogCyberslacking** [comment]
yes,  you can modify the coode in python/jnerf/models/samplers/density_grid_sampler/op_header/mark_untrained_density_grid.h
